diff --git a/node_modules/hexo-douban/lib/http.js b/node_modules/hexo-douban/lib/http.js
index d17af0a..05e7285 100644
--- a/node_modules/hexo-douban/lib/http.js
+++ b/node_modules/hexo-douban/lib/http.js
@@ -29,6 +29,10 @@ module.exports.fetchBook = async function (id, action, referer, timeout) {
     for (let i in bookResult.comment) {
         let comment = bookResult.comment[i]
         let metas = []
+        if (/^(\d+)-(\d+)-(\d+)$/.test(comment.comment)) {
+            comment.mark_date = comment.comment
+            comment.comment = ''
+        }
         if (comment.item.author) {
             metas.push(comment.item.author)
         }
@@ -180,15 +184,19 @@ module.exports.fetchMovie = async function (id, action, referer, timeout) {
     for (let i in movieResult.comment) {
         let comment = movieResult.comment[i]
         let metas = []
-        if (comment.item.style) {
-            metas.push(comment.item.style)
-        }
-        if (comment.item.director) {
-            metas.push(comment.item.director)
-        }
-        if (comment.item.writer) {
-            metas.push(comment.item.writer)
-        }
+        if (/^(\d+)-(\d+)-(\d+)$/.test(comment.comment)) {
+            comment.mark_date = comment.comment
+            comment.comment = ''
+        }
+        // if (comment.item.style) {
+        //     metas.push(comment.item.style)
+        // }
+        // if (comment.item.director) {
+        //     metas.push(comment.item.director)
+        // }
+        // if (comment.item.writer) {
+        //     metas.push(comment.item.writer)
+        // }
         if (comment.item.actor) {
             metas.push(comment.item.actor)
         }
@@ -210,6 +218,7 @@ module.exports.fetchMovie = async function (id, action, referer, timeout) {
             alt: "https://movie.douban.com/subject/" + comment.item.douban_id + "/",
             image: comment.item.thumbnail,
             meta: metas.join(" / "),
+            style: comment.item.style,
             rating: rating.join(" / "),
             comment: comment.comment
         })
diff --git a/node_modules/hexo-douban/lib/templates/index.css b/node_modules/hexo-douban/lib/templates/index.css
index 633a3e5..cc73253 100644
--- a/node_modules/hexo-douban/lib/templates/index.css
+++ b/node_modules/hexo-douban/lib/templates/index.css
@@ -10,7 +10,7 @@
 }
 
 .hexo-douban-tab-active {
-    background: #657b83;
+    background: #0d0d0d;
     color: #fff !important;
 }
 
@@ -74,7 +74,7 @@
 }
 
 .hexo-douban-button:hover {
-    background: #657b83;
+    background: #0d0d0d;
     color: #fff !important;
     opacity: 1;
 }
diff --git a/node_modules/hexo-douban/lib/templates/index.ejs b/node_modules/hexo-douban/lib/templates/index.ejs
index 268b302..dc7df67 100644
--- a/node_modules/hexo-douban/lib/templates/index.ejs
+++ b/node_modules/hexo-douban/lib/templates/index.ejs
@@ -90,7 +90,7 @@
             let tabCollect = document.querySelector(".hexo-douban-tab-collect");
             tabCollect.onclick = () => tabClick(tabCollect, 'collect');
 
-            tabCollect.click();
+            tabDo.click();
         })();
 
         // ======== render ========
@@ -136,6 +136,10 @@
             link.textContent = data.title
             title.append(link)
 
+            let style = document.createElement('div')
+            style.className = 'hexo-douban-comment'
+            style.textContent = data.style
+
             let meta = document.createElement('div')
             meta.className = 'hexo-douban-meta'
             meta.textContent = data.meta
@@ -149,7 +153,7 @@
             comment.className = 'hexo-douban-comment'
             comment.textContent = data.comment
 
-            info.append(title, meta, rating, comment)
+            info.append(title, style, meta, rating, comment)
 
             item.append(picture, info)
             return item
